content/changelog/2024-09-03-gitlab-components-review-apps.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Version 2.0 of our GitLab Component for review applications is available"
+date: 2024-09-03
+tags:
+  - gitlab
+  - component
+authors:
+  - name: David Legrand
+    link: https://github.com/davlgd
+    image: https://github.com/davlgd.png?size=40
+  - name: Julia March
+    link: https://github.com/juliamrch
+    image: https://github.com/juliamrch.png?size=40
+description: Simplify your GitLab workflows
+excludeSearch: true
+---
+
+Some months ago, we released [a new GitHub Component](https://gitlab.com/explore/catalog/CleverCloud/clever-cloud-pipeline) to deploy applications from GitLab to Clever Cloud. Version 2.0 is available with variables injection from GitLab CI/CD and automatic releases. You can now use the `latest` tag and input deploy have been removed to prevent users to script the environment variables injection before deploying the app.
```

Comment:
```suggestion
Some months ago, Clever Cloud released [a new GitHub Component](https://gitlab.com/explore/catalog/CleverCloud/clever-cloud-pipeline) to deploy applications from GitLab to Clever Cloud. Version 2.0 is available with variables injection from GitLab CI/CD and automatic releases. You can now use the `latest` tag and input deploy have been removed to prevent users to script the environment variables injection before deploying the app.
```